pkg/store/postings_codec.go

```diff
@@ -25,14 +27,168 @@ import (
 // significantly (to about 20% of original), snappy then halves it to ~10% of the original.
 
 const (
-	codecHeaderSnappy = "dvs" // As in "diff+varint+snappy".
+	codecHeaderSnappy         = "dvs" // As in "diff+varint+snappy".
+	codecHeaderStreamedSnappy = "dss" // As in "diffvarint+streamed snappy".
 )
 
+func getDecodingFunction(input []byte) func([]byte) (closeablePostings, error) {
+	if isDiffVarintSnappyEncodedPostings(input) {
```

Comment:
I assume eventually this won't be used as everything will be streamed-encoded? Do we have a plan to remove this maybe in couple of versions?

Reply:
Maybe we could remove it for v1.0? What do you think would be reasonable here?